vllm/lora/punica.py

```diff
@@ -131,9 +132,9 @@ def convert_mapping(
     if long_lora_context:
         assert long_lora_offsets is not None
         indices_list.append(long_lora_offsets)
-    indices = torch.tensor(indices_list, dtype=torch.long, device="cuda")
+    indices = torch.tensor(indices_list, dtype=torch.long, device=get_device())
```

Comment:
```suggestion
    device = 'hpu' if current_platform.is_hpu() else 'cuda'
    indices = torch.tensor(indices_list, dtype=torch.long, device=device)
```
Then use `device` instead of `get_device()` in the remainder of the file. For getting `current_platform`, add `from vllm.platforms import current_platform` at the top of the file.

Reply:
We are using get_device; to localize this if check; as get_device is used in many places in lora/models.py, lora/punica.py. This way the code looks clean